src/lib/utils.js

```diff
@@ -54,14 +54,7 @@ export const trimAndFormatPath = (
     );
   }
   // can't fit dirname, but can fit trimmed basename
-  return slash(
-    chalk.bold(
-      `${TRIMMING_DOTS}${basename.slice(
-        basename.length - maxLength - 4,
-        basename.length,
-      )}`,
-    ),
-  );
+  return slash(chalk.bold(`${TRIMMING_DOTS}${basename.slice(-maxLength + 3)}`));
```

Comment:
While working on my previous PR I've noticed this issue. It basically was caused by the fact that `maxLength` was bigger than `basename.length` so a negative starting index has been calculated. We don't need to provide both start + end indexes as we always want to slice "till the end" - calculating just a starting index as a negative one simplifies things.